src/cljs/commiteth/common.cljs

```diff
@@ -30,9 +31,22 @@
 (defn issue-url [owner repo number]
   (str "https://github.com/" owner "/" repo "/issues/" number))
 
+(defn scroll-div [container-element]
```

Comment:
Why exactly do we need this no-op component if we can just perform this behaviour in the list components?

Reply:
The idea was that list components might get updated during any page content changes, not just page number changes.

Reply:
I've been thinking about when the scrolling should happen, and it is a little tricky.

The behaviour we are looking for is for the scrolling to trigger when the user navigates to a different page.

And it should not trigger in some other cases:

* As you said, when the list component updates for other reasons.
* And also when the page number changes for other reasons that are not the user navigating. For example, when changing the page, the page number is restarted, but the scrolling should not happen there. Or when the filters change, the page number is restarted as well, and the scrolling should not happen there either.

In order to get this I think the best thing is to pass the ref of the list to the children components so that the buttons where the clicks occur in the navigator have it, and trigger there the scrolling.

I think that is better because we trigger the scrolling exactly where we want it to happen, that is, when the user interacts with the navigator, and don't try to guess that behaviour from state changes in the page number.

Reply:
Thanks for explaining! Just pushed an update.